src/test/java/core/basesyntax/service/impl/FruitShopServiceImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.operations.BalanceOperation;
+import core.basesyntax.operations.OperationHandler;
+import core.basesyntax.service.FruitShopService;
+import core.basesyntax.service.ReportCreator;
+import core.basesyntax.service.impl.exception.InvalidDataException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FruitShopServiceImplTest {
+    private FruitShopService fruitShopService;
+    private OperationHandler operationHandler;
+    private ReportCreator reportCreator;
+    private FruitTransaction validFruitTransaction;
+    private FruitTransaction inValidFruitTransaction;
+
+    @BeforeEach
+    void setUp() {
+        fruitShopService = new FruitShopServiceImpl();
+        operationHandler = new BalanceOperation();
+        reportCreator = new ReportCreatorImpl();
+        validFruitTransaction =
```

Comment:
are you sure that you want to create same fruitTransaction each time you run test?